vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -463,11 +463,9 @@ winrt::IPropertyValue TouchEventHandler::TestHit(
         return resTag;
 
       if (isHit) {
-        if (auto fe = el.try_as<xaml::FrameworkElement>()) {
```

Comment:
Not necessary here, since RNW is the one creating the Spans/Runs and their tags.